cypress/integration/forms/buttons.spec.js

```diff
@@ -0,0 +1,162 @@
+describe('Buttons', () => {
+  before(() => {
+    cy.visit('/buttons', { failOnStatusCode: false });
+    cy.injectAxe();
+    cy.get('.page-loader').should('not.exist', { timeout: 20000 });
+  });
+
+  describe('button', () => {
+    beforeEach(() => {
+      cy.get('[sectiontitle="Buttons"]').as('SUT');
+    });
+
+    it('has no detectable a11y violations on load', () => {
+      cy.get('@SUT').within(() => {
+        cy.get('.ngx-section-content button').then($el => {
+          cy.checkA11y($el, {
+            rules: {
+              'color-contrast': { enabled: false }
```

Comment:
question: this is waiting on design to come through with updated designs with appropriate color designs right? if so probably should leave a TODO to make sure we enable the contrast check after those designs are in.